tensorlayer/prepro.py

```diff
@@ -1719,8 +1719,8 @@ def transform_matrix_offset_center(matrix, x, y):
     - See ``tl.prepro.rotation``, ``tl.prepro.shear``, ``tl.prepro.zoom``.
 
     """
-    o_x = float(x) / 2 + 0.5
-    o_y = float(y) / 2 + 0.5
+    o_x = (x - 1) / 2
```

Comment:
In python2, it doesn't work.
Still need `(x - 1) / 2.0`

Reply:
Ah, it's my mistake!